tests/dummy/app/templates/demos/sl-menu.hbs

```diff
@@ -60,7 +60,7 @@
         <ul>
             <li><strong>pages</strong> - An array of nodes that will be embedded under the <code>label</code> attribute as a sub-menu.</li>
             <li><strong>label</strong> - The label of the menu itself. This should be <em>null</em> for top level menus and will provide the name of the sub-menu for sub-menu lists.</li>
-            <li><strong>classNames</strong> - An array of class names to be added to the element's <code>class</code> attribute.</li>
+            <li><strong>className</strong> - An string of classes to be added to the element's <code>class</code> attribute.</li>
```

Comment:
Prefer that these be specified as an array rather than a text string. It will be easier to dynamically manipulate an array of options passed to this component than a text string.